nuxt.config.js

```diff
@@ -379,10 +379,17 @@ export default defineNuxtConfig({
         query: { compact: true },
       })
 
+      config.module.rules.push({
```

Comment:
what's this? transpiler?

Reply:
Yup otherwise it wont compile. Chceck this gh action:

https://github.com/kodadot/nft-gallery/actions/runs/4681018114/jobs/8293083530#step:4:62